persist/fs/seek.go

```diff
@@ -221,14 +213,39 @@ func (s *seeker) Open(namespace ts.ID, shard uint32, blockStart time.Time) error
 		return err
 	}
 
+	s.indexLookup, err = newNearestIndexOffsetLookupFromSummariesFile(
+		summariesFd,
+		s.summariesFdWithDigest,
+		s.expectedSummariesDigest,
+		s.decoder,
+		int(s.summariesInfo.Summaries),
+	)
+	if err != nil {
+		s.Close()
+		return err
+	}
+
 	if !s.keepUnreadBuf {
```

Comment:
Do we need this unreadbuf anymore?  Can we kill this whole concept?

Reply:
Looks like its used to read the info file, so not sure if this P.R affects that